File: Програмування/С++ ООП/Project13.12.18/Project13.12.18/Person.cpp
#include"Person.h"

void Person::Show()
{
	cout << "name:" << name << endl << "surname:" << surname << endl;
}
void Person::SetData(string name,string surname)
{

	cout << "Enter name:";
	cin >> name;
	cout << "Enter surname:";
	cin >> surname;
	this->name = name;
	this->surname = surname;
}

File: Програмування/WinForms/CV/CV/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CV
{
    class Person
    {
        public string Name { get; }
        public string SurName { get; }
        public int Age { get;  }
        public string Gouder { get; }
        public string City { get; }
        public List<string> reasons = new List<string>();
        public Person(string n,string s,int a,string g,string c,List<string> r)
        {
            Name = n;
            SurName = s;
            Age = a;
            Gouder = g;
            City = c;
            reasons = r;
        }
    }
}
